datus/tools/func_tool/context_search.py

```
@@ -237,3 +225,52 @@ def search_reference_sql(
+                    - 'sql'
+                    - 'comment'
+                    - 'tags'
+                    - 'summary'
+                    - 'file_path'
+        """
+        try:
+            result = self.reference_sql_store.get_reference_sql_detail(subject_path=subject_path, name=name)
+            if len(result) > 0:
+                return FuncToolResult(success=1, error=None, result=result[0])
+            return FuncToolResult(success=0, error="No matched result", result=None)
+        except Exception as e:
```

Ruff (0.14.10)

Do not catch blind exception: `Exception`

(BLE001)